elementary/clients/gcs/client.py

```diff
@@ -2,10 +2,12 @@
 from typing import Optional
 
 import google
+from google.auth.credentials import Credentials
 from google.cloud import storage
+from google.oauth2 import service_account
```

Comment:
Should we add any of these imports to setup.py?

Reply:
No, they're all included in `google-cloud-storage`.
You can tell this is the case due to the tests working properly.